code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -106,15 +106,20 @@
 	item_state = "laceups"
 
 /obj/item/clothing/shoes/galoshes
-	desc = "Rubber boots"
 	name = "galoshes"
+	desc = "Rubber boots"
 	icon_state = "galoshes"
 	permeability_coefficient = 0.05
 	flags = NOSLIP
 	slowdown = SHOES_SLOWDOWN+1
 	species_fit = list(VOX_SHAPED)
 	heat_conductivity = INS_SHOE_HEAT_CONDUCTIVITY
 
+/obj/item/clothing/shoes/galoshes/vault/broken
```

Comment:
What if someone wants to put them somewhere that *isn't* a vault at some point?

People don't need to know that these have been placed in a vault. All they need to know is that they're a child of galoshes with a different sprite and the slip resistance removed. They can be used for whatever anyone wants to use them for.